daemon/lib/swaps/40swap.go

```diff
@@ -0,0 +1,88 @@
+package swaps
```

Comment:
same come as in other PR's, i would avoid using `lib` and flatten it to `swaps/40swap.go`

Reply:
done 🏁